src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -35,6 +35,12 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+// @todo #18:30min Impediment: #103 must be fixed before LCOM5 is tested
```

Comment:
@llorllale Same here:

1) these 2 puzzles in the javadoc

2) As in the other PR, shouldn't it return something else than ``NaN`` until 103 is solved? Maybe -1?

Reply:
@amihaiemil my replies to these two concerns in the other PR (#113) apply here. Please let me know.

Reply:
@llorllale ok